jooby-neo4j/src/main/java/org/jooby/neo4j/Neo4j.java

```diff
@@ -0,0 +1,259 @@
+ *   use(new Neo4j("db1"));
+ *   use(new Neo4j("db2"));
+ *
+ *   get("/", req {@literal ->} {
+ *     BoltDBAccess client1 = req.require("mydb1", BoltDBAccess.class);
+ *     // work with mydb1
+ *     BoltDBAccess client2 = req.require("mydb2", BoltDBAccess.class);
+ *     // work with mydb1
+ *   });
+ * }
+ * </pre>
+ *
+ * @author sbcd90
+ * @since 1.1.1
+ */
+public class Neo4j implements Jooby.Module {
+
+  private final String db;
+
+  private BiConsumer<Properties, Config> properties;
+
+  /**
+   * Creates a new {@link Neo4j} module.
+   *
+   * @param db Name of the property with the connection info.
+   */
+  public Neo4j(final String db) {
+    this.db = requireNonNull(db, "A neo4j bolt configuration is required");
+  }
+
+  /**
+   * Creates a new {@link Neo4j} using the default property: <code>db</code>.
+   */
+  public Neo4j() {
+    this("db");
+  }
+
+  @Override
+  public void configure(Env env, Config config, Binder binder) throws Throwable {
+    configure(env, config, binder, (remoteClient) -> {});
+  }
+
+  protected void configure(final Env env, final Config config, final Binder binder,
+                           final Consumer<BoltDBAccess> callback) {
+    Properties properties = options(neo4j(config));
+
+    if (nonNull(this.properties)) {
+      this.properties.accept(properties, config);
+    }
+
+    String dbUser = nonNull(properties.getProperty("username")) ? properties.getProperty("username"): "";
+    String dbPassword = nonNull(properties.getProperty("password")) ? properties.getProperty("password"): "";
+
+    BoltDBAccess remoteClient;
+    if (dbUser.isEmpty() && dbPassword.isEmpty()) {
+      remoteClient = (BoltDBAccess) DBAccessFactory.createDBAccess(DBType.REMOTE, properties);
+    } else {
+      remoteClient = (BoltDBAccess) DBAccessFactory.createDBAccess(DBType.REMOTE, properties, dbUser, dbPassword);
+    }
+    checkArgument(remoteClient.getSession() != null,
+      "Cannot connect to Database at: " + properties.getProperty(DBProperties.SERVER_ROOT_URI));
+    String database = properties.getProperty(DBProperties.SERVER_ROOT_URI);
+
+    ServiceKey serviceKey = env.serviceKey();
+    serviceKey.generate(BoltDBAccess.class, database, k -> binder.bind(k).toInstance(remoteClient));
+
+    env.onStop(remoteClient::close);
+
+    callback.accept(remoteClient);
+  }
+
+  /**
+   * Set a properties callback.
+   *
+   * <pre>
+   * {
+   *   use(new Neo4j()
+   *     .properties((properties, config) {@literal ->} {
+   *       properties.put("arrayBlockSize", "120);
+   *     })
+   *   );
+   * }
+   * </pre>
+   *
+   * @param properties Configure callback.
+   * @return This module
+   */
+  public Neo4j properties(final BiConsumer<Properties, Config> properties) {
+    this.properties = requireNonNull(properties, "properties callback is required");
+    return this;
+  }
+
+  @Override
+  public Config config() {
+    return ConfigFactory.parseResources(Neo4j.class, "org.jooby.neo4j/neo4j.conf");
```

Comment:
does it work? It should be: `ConfigFactory.parseResources(Neo4j.class, "neo4j.conf");`